ovs/extensions/healthcheck/expose_to_cli.py

```diff
@@ -0,0 +1,324 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+import ast
+import imp
+import os
+import inspect
+from datetime import datetime, timedelta
+from ovs.extensions.healthcheck.result import HCResults
+from ovs.log.log_handler import LogHandler
+
+
+# class decorator
+# noinspection PyPep8Naming
+class expose_to_cli(object):
+    def __init__(self, module_name=None, method_name=None):
+        if module_name and method_name:
+            self.module_name = module_name
```

Comment:
It would be better to add a property like `expose_to_cli` that contains a dict with all info. Names like `module_name` are too generic.